test/e2e/specs/editor/various/block-mover.spec.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'block mover', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should show block mover when more than one block exists', async ( {
+		editor,
+		page,
+	} ) => {
+		// Create a two blocks on the page.
+		await page.click( '.block-editor-default-block-appender' );
+		await page.keyboard.type( 'First Paragraph' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( 'Second Paragraph' );
```

Comment:
I think it would be good to simplify this part with something like:
```js
await editor.insertBlock( { name: 'core/paragraph', attributes: { content: 'First Paragraph' } );
await editor.insertBlock( { name: 'core/paragraph', attributes: { content: 'Second Paragraph' } );

Reply:
Worked on the feedbacks